src/install/aim/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,129 @@
+            title="Enable with config file"
+        >
+
+    ```ini
+    ai_monitoring.enabled = true
+    ```   
+        </Collapser>
+        <Collapser
+            id="config-enviro"
+            title="Enable with environment variables"
+        >
+
+    ```bash
+    NEW_RELIC_AI_MONITORING_ENABLED=true
+    ```
+
+        </Collapser>
+    </CollapserGroup>
+
+2. These configurations adjust how the agent captures and sends AI data. They let you limit the number of events or spans the agent collects, increase or decrease message limits, or enable logging.  
+
+    <CollapserGroup>
+        <Collapser
+            id="py-event-limit"
+            title="ELEPHANT"
+        >
+
+        One instance of the Python agent sends a maximum of 100,000 events per minute. Once the agent hits that maximum, it starts to sample event data, giving a general representation of events across the harvest cycle. 
+
+        Set limit with config file: 
+
+            ```ini
+                event_harvest_config.harvest_limits.custom_event_data
+            ```
+
+        Set limit with environment variables: 
+
+            ```bash
+                NEW_RELIC_ML_INSIGHTS_EVENTS_MAX_SAMPLES_STORED
+            ```
+
+        * Setting this value to the maximum value (100k) means that no sampling occurs. However, ingest and memory storage will go up.
+        * Setting this to a lower value (1k or less) prompts the agent to sample, depending on the volume of your app. If you're concerned with cost or need less data in New Relic, setting a lower value is a good option. 
+
+        </Collapser>
+        <Collapser
+            id="py-event-harvest"
+            title="ELEPHANT"
+        >
+
+        Set a limit for how many sample spans the agent records. Spans capture when an operation occurred in your app code, how long it took to complete, and other operations it relates to. 
+
+        Set limit with config file:
+
+        ```ini
+            event_harvest_config.harvest_limits.span_event_data
+        ```              
+
+        Set limit with environment variables:
+
+        ```bash
+            ELEPHANT
+        ```
+
+        </Collapser>
+        <Collapser
+            id="py-spans-limit"
+            title="ELEPHANT"
+        >
+
+        Limit the number of spans the agent samples
+
+            ```ini
+                custom_insights_events.max_samples_stored
+            ```
+                
+        </Collapser>
+        <Collapser
+            id="py-attribute-limits"
+            title="Increase attribute limits on AI messages"
+        >
+
+        Increases the attribute limit on messages in LLM events. 
+
+            * When AIM ingests large messages, shortened messages can lose context. 
+            * The maximum value is 4,095.
+
+        ```ini
+            custom_insights_events.max_attribute_value
+        ```  
+                        
+        </Collapser>
+        <Collapser
+            id="enable-logs"
```

Comment:
What is the difference btwn enabling app logging vs enabling app forwarding? What environment set up will customers have that makes one more beneficial over the other?

Reply:
I feel like we might be getting off in the weeds by mentioning any sort of logging setting here. Logging is not specific to large language model instrumentation. Maybe just a quick note about connecting logs to NR data and then provide them a link to our logs in context docs: https://docs.newrelic.com/docs/logs/logs-context/get-started-logs-context/. Forwarding is explained in those docs but just to answer your question here. Forwarding actually sends the logs to NR via the agent as opposed to letting another logging tool record/push up the logs:

> For example, if you want to have an APM agent add New Relic metadata, you can use the log decoration feature, while at the same time disabling log forwarding and instead using another logging agent (for example, our infrastructure agent, or a third-party logging agent).